src/components/TotalAmount/index.tsx

```diff
@@ -6,13 +6,14 @@ import './TotalAmount.scss';
 
 interface ComponentProps {
   amount: number;
-  className?: string;
   title: string;
+  className?: string;
+  dataTestId?: string;
 }
 
-const TotalAmount: FC<ComponentProps> = ({amount, className, title}) => {
+const TotalAmount: FC<ComponentProps> = ({amount, className, title, dataTestId = 'TotalAmount'}) => {
   return (
-    <div className={clsx('TotalAmount', className)}>
+    <div data-testid={dataTestId} className={clsx('TotalAmount', className)}>
       <div className={clsx('TotalAmount__title', {...bemify(className, '__title')})}>{title}</div>
       <div className={clsx('TotalAmount__amount', {...bemify(className, '__amount')})}>{amount.toLocaleString()}</div>
```

Comment:
can you add a test that will test for this `amount.toLocaleString()`? You can read about that here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString

Reply:
sure can do, 
@angle943 do we want to had an prop for testing locale or just adding a test for locale?

Reply:
ah i was thinking about just checking for the `commas` to make it more readable